apps/web/src/services/asset-inventory/collector/create-collector/CreateCollectorPage.vue

```diff
@@ -45,6 +54,7 @@ import { ASSET_INVENTORY_ROUTE } from '@/services/asset-inventory/route-config';
 
 const state = reactive({
     step: 1,
+    isDeleteModalVisible: false,
```

Comment:
quibble: I think `deleteModalVisible` would be more suitable for our convention.